terminaltables.py

```diff
@@ -22,21 +22,56 @@
 import re
 import struct
 import sys
+import unicodedata
+
+PY2 = sys.version_info[0] == 2
 
 try:
     import fcntl
     import termios
-except ImportError:
+except ImportError:  # pragma: no cover
     fcntl = None
     termios = None
 
-from colorclass import _WindowsCSBI
+from colorclass import _WindowsCSBI, Color
 
 __author__ = '@Robpol86'
 __license__ = 'MIT'
 __version__ = '1.2.0'
 
 
+if PY2:  # pragma: no cover
+    def _to_unicode(string):
+        if isinstance(string, str):
+            return string.decode('u8')
+        return string
+else:  # pragma: no cover
+    _to_unicode = lambda x: x
+
+
+def _get_width(string):
+    """Get the real width of unicode string.
+
+    Positional arguments:
+    string -- unicode string.
+
+    Returns:
+    String width.
+    """
+    if isinstance(string, Color):
```

Comment:
I think it won't be so easy since something like sys.stdout.write() iterates the Color instance and that expects the color version. You're free to open an issue/pr in colorclass. Maybe instead you can just use hasattr() or getattr() for value_no_colors?